### PR TITLE
[PRM-224] - added reasonableExcuses form error

### PR DESCRIPTION
**Pull request by BackRowOla**

### CODE REVIEW

**Comment by BackRowOla** [comment]
Screenshot 2021-05-17 at 15 23 25

File: conf/messages [hunk 1]
```diff
@@ -5,6 +5,8 @@ common.pageTitle = {0} - {1} - GOV.UK
 
 app.common.back = Back
 
+error.summary.title = There is a problem
```
Comment (LewisMitchell) [inline conf/messages:8]: As part of having an error on the page - we need to modify the page title to prefix `Error: `
I've done this in https://github.com/hmrc/penalties-appeals-frontend/commit/42792af03390c359b9bb8f8f70cf34cd1dde5bd5

File: conf/messages [hunk 2]
```diff
@@ -59,6 +61,7 @@ reasonableExcuses.technicalIssuesReason = Technology issues
 reasonableExcuses.otherReason = The reason does not fit into any of the other categories
 reasonableExcuses.breakerText = or
 reasonableExcuses.externalGuidanceLinkText = Read more about reasonable excuses
+reasonableExcuses.error.required = Select the reason for missing the VAT deadline
```
Comment (LewisMitchell) [inline conf/messages:64]: Could you copy these messages over to messages.cy

File: test/controllers/ReasonableExcuseControllerSpec.scala [hunk 3]
```diff
@@ -190,6 +190,25 @@ class ReasonableExcuseControllerSpec extends SpecBase {
...
+        val result = controller.onSubmit()(fakeRequestWithCorrectKeys.withJsonBody(
+          Json.parse(
+            """
+              |{
+              |   "value": ""
+              |}
+              |""".stripMargin
+          )
+        ))
+        status(result) shouldBe BAD_REQUEST
```
Comment (LewisMitchell) [inline test/controllers/ReasonableExcuseControllerSpec.scala:210]: Could you assert what's on the page? Like making sure `"There is a problem"` and `"Select the reason for missing the VAT deadline"` is on the page?
I know it's not really the controllers job - but it saves doing it in the view.

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Looks good to me, just a few changes.

**Comment by LewisMitchell** [comment]
image
Revised.

**Comment by LewisMitchell** [comment]
Looks good, just needs a message copying to messages.cy

```
# Errors
# ----------------------------------------------------------
error.prefix = Error:
```

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Just the messages.

**Review by LewisMitchell — APPROVED** [review]
Fine by me just waiting for #12

**Review by AndrewRButcher — APPROVED** [review]

**Pull request merged**